web/src/user_card_popover.js

```diff
@@ -3,7 +3,6 @@ import {parseISO} from "date-fns";
 import $ from "jquery";
 import tippy from "tippy.js";
 
-import render_no_arrow_popover from "../templates/no_arrow_popover.hbs";
```

Comment:
`web/templates/no_arrow_popover.hbs` should be deleted since it's now unused.

Reply:
Yeah, I overlooked that deletion. I'll fix it.